templates/field/field--media--oe-media-iframe--iframe--oe-theme-main-content.html.twig

```diff
@@ -0,0 +1,18 @@
+{#
+/**
+ * @file
+ * Theme override for iframe fields on media items.
+ */
+#}
+{% for item in items %}
+  {% if ratio is empty %}
+    <div class="media-iframe-wrapper">
+      <div class="ecl-media-container__media">{{ item.content }}</div>
+    </div>
+  {% else %}
+    {% include '@ecl-twig/media-container' with {
+      'embedded_media': item.content,
+      'ratio': ratio|replace({'_': '-'}),
+    } %}
+  {% endif %}
```

Comment:
```suggestion
  {% include '@ecl-twig/media-container' with {
    'embedded_media': item.content,
    'ratio': ratio|replace({'_': '-'}),
    'extra_classes': (ratio == 'custom') ? 'ecl-media-container--custom-ratio' : '',
  } %}
```

Reply:
Passing a non-existing ratio such as `custom` will prevent the ratio to apply. But in this way we can reuse all the code from the ECL component. To shrink the container to the size of the iframe, we can use inline-block or some other suggestion from the frontend team (good ol' float:left plus clear on the pseudo after element).

Reply:
Fixed